ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
Sorry, I didn't get a notification for some reason...
We would leak in the real application. But this is a unit test which doesn't use real connections. There is nothing to leak.
I'm not sure what you mean by close not being executed. Why would isClose execute it? You should check for closeImpl execution in closeConnectionInAfterCompletion test.